### ISSUE TITLE
Negative card drops and games left

### ISSUE THREAD

**Issue by Keavon** [jshackles/idle_master#64]
I idled two games and minimized Idle Master to the system tray. Then I opened it several hours later and found this:
![](http://i.imgur.com/5Pw8wpk.png)
![](http://i.imgur.com/XtaSVO2.png)
![](http://i.imgur.com/SqaIlvL.png)
![](http://i.imgur.com/kE4aAXL.png)

The number is decreasing by 1 about every 3 seconds. It started at about -260.

**Comment by m3Lith** [comment]
So after finishing to idle my games I've noticed this too. By quickly inspecting the source I'd say the bug is caused by [this](https://github.com/jshackles/idle_master/blob/master/.NET%20%28Windows%29/IdleMaster/frmMain.cs#L625) sequence. Mainly, the thing that happens after there're no drops remaing, is `checkCardDrops` calls `idleComplete()`, which only does `tmrCardDropCheck.Enabled = false;` and prints `"Idling complete"`. However, the line after `checkCardDrops`, which I linked, enables the timer, thus repeating the process over and over again, and as such decreasing the counters into negative numbers by -1 every tick. 

One solution would be to simply make `checkCardDrops` return `true` if there're any drops remaining at all and only then line 625 should be executed.
